drag_and_drop_v2/drag_and_drop_v2.py

```diff
@@ -432,39 +481,37 @@ def _get_zone_by_uid(self, uid):
             if zone["uid"] == uid:
                 return zone
 
-    def _get_grade(self):
+    def _get_item_stats(self):
         """
-        Returns the student's grade for this block.
+        Returns a 3-tuple representing the number of correct items placed, the number of completed
+        items, right or wrong, and the total number of items that must be completed.
```

Comment:
@haikuginger This docstring is outdated now, could you please update it? To keep the terminology consistent I think it would be good to talk about "placed items" instead of "completed items" in the context of this method:

> Returns a tuple representing the number of correctly-placed items, and the total number of items that must be placed on the board (non-decoy items).